homeassistant/components/ollama/const.py

```diff
@@ -11,6 +11,11 @@
 KEEP_ALIVE_FOREVER = -1
 DEFAULT_TIMEOUT = 5.0  # seconds
 
+CONF_NUM_CTX = "num_ctx"
+DEFAULT_NUM_CTX = 8192
+MIN_NUM_CTX = 2048
+MAX_NUM_CTX = 65536
```

Comment:
Why is max 64K and not 128K, which would be 131072?

Reply:
Agreed, intended to set to highest context window for llama3.1 (128k) and forgot when copying.